packages/performance/src/services/remote_config_service.ts

```diff
@@ -271,9 +271,6 @@ function shouldLogAfterSampling(samplingRate: number): boolean {
 /**
  * True if event should be sent to Fl transport endpoint rather than CC transport endpoint.
  * rolloutPercent is in range [0.0, 100.0].
- * @param {string} iid Installation ID which identifies a web app installed on client.
```

Comment:
hm, actually I suggested to only remove the type information in the comment, that is e.g. `{string}`, and we should keep the rest of the comments.

Reply:
Thanks for the callout! I overlooked your message about `parameter type` instead of `parameter`.